features/add_categories.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Add and Edit Categories
```

Comment:
Maybe a better name would be "manage categories" so if something other than add/edit came up if would be covered